src/java/main/org/apache/zookeeper/server/quorum/Follower.java

```diff
@@ -135,6 +135,8 @@ protected void processPacket(QuorumPacket qp) throws IOException{
         case Leader.SYNC:
             fzk.sync();
             break;
+        default:
+            LOG.error("Invalid packet type received by Observer: " + qp.getType());
```

Comment:
Can we change log message like below to improve readability.
`LOG.error("Invalid packet type: {} received by Observer", qp.getType());`